packages/types/api.d.ts

```diff
@@ -2051,7 +2051,7 @@ export interface IParserErrorMessageProvider {
     expectedPathsPerAlt: TokenType[][][]
     actual: IToken[]
     previous: IToken
-    customUserDescription: string
+    customUserDescription: string | undefined
```

Comment:
most (all?) other public APIs use **optional** config properties, why should this be `| undefined`?